news/12319-add-support-for-64-bit-riscv

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* Add support for the 64-bit RISC-V architecture on Linux (#12319)
```

Comment:
This should mention the update of the dependency

Reply:
Added an addition item for the py-cpuinfo upgrade.